src/Renderer/PhpRenderer.php

```diff
@@ -479,6 +471,7 @@ public function render($nameOrModel, $values = null)
                 ob_start();
                 $includeReturn   = include $this->__file;
                 $this->__content = ob_get_clean();
+                assert($this->__content !== false);
```

Comment:
Can't happen

Reply:
@Ocramius 
Sure?
> Returns the contents of the active output buffer on success or [false](https://www.php.net/manual/en/reserved.constants.php#constant.false) on failure.

https://www.php.net/manual/function.ob-get-clean.php

Reply:
This was just for Psalm because of the documented `string|false` return type. I'll revert and expand the baseline 👍

Reply:
With the `string` property type, we'd get a `TypeError` before the `false` could proliferate